components/item/MetadataField.vue

```diff
@@ -33,13 +33,29 @@
             />
           </li>
         </template>
+        <template v-else-if="typeof fieldData.url !== 'undefined'">
```

Comment:
Would it work to just say `<template v-else-if="fieldData.url">`?

Reply:
yes that does appear to suffice